images/README.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
why a readme here?

Reply:
To display a either a help page which is automatically generated by Github, or a link to docs.fortinet.com connector page, so ppl will get help how to install/use the connector